index/models.py

```diff
@@ -122,15 +122,14 @@ def add_info_other(self, info):
 class Petition(models.Model):
     title = models.CharField(max_length=200)  # Title of petition
     description = models.CharField(max_length=4000)  # The paragraph description
-    ID = models.IntegerField(primary_key=True)
     archived = models.BooleanField(default=False)
     hidden = models.BooleanField(default=False)
 
     created_date = models.DateTimeField(
         db_index=True, default=timezone.now
     )  # Files the date created
     expected_sig = models.IntegerField(
```

Comment:
Yes I agree now, see @kochman's comment and my response above. I think we could still have the `expected_sig` read from an env variable, e.g. `default=os.environ.get("EXPECTED_SIG")`. This way if the number changes, we don't have to modify the source code and redeploy, we just have to change the environment variable.